### DISCUSSION TITLE
LTSP in Vlan PFsense (almost working) ipxe issue

### DISCUSSION THREAD

**Discussion by doguibnu** [ipxe/ipxe#774] [General]
Hello!
We are trying to boot up ltsp by Vlan on pfsense. 

So I would like to show some screenshots about pfsense

the vlan
![01-vlan-nome](https://user-images.githubusercontent.com/38897311/195860084-0b81789b-bf85-418b-bdea-6af368fb28ad.png)

Ports open vlan:
![02-vlan-config](https://user-images.githubusercontent.com/38897311/195860170-d2fd542a-f5a1-4277-94f3-0718fce9dc8a.png)

TFTP-proxy enable:
![03-tftp-enable](https://user-images.githubusercontent.com/38897311/195860303-c7d90c99-6301-4170-a4a2-75d9fe468300.png)

DHCP Vlan enable:
![04-dhcp-server-vlan712](https://user-images.githubusercontent.com/38897311/195860460-dc475cd3-e1c3-46db-8a01-e1cc6f140b68.png)

TFTP - server config
![5-1-tftp-vlan2](https://user-images.githubusercontent.com/38897311/195860713-25f9b4ed-0d1d-4627-9f6c-1d607a53b5e4.png)

I follow the official wiki to install [ltsp server ](https://ltsp.org/docs/installation/) without dhcp, because vlan has enable DHCP server. 

After install the LTSP server, the clients can not boot from vlan. It shows me this screeenshot:

![ultima_20221013_110122](https://user-images.githubusercontent.com/38897311/195862139-3e3a5136-a3af-4062-9329-80edc46ca587.jpg)
And stay  in looping.

Please, someone can help me about this issue

Thank you1

**Comment by NiKiZe** [comment]
Undionly is unlikely to support vlan.
And there is nothing that automatically makes ipxe use vlans.

Now I think this whole thing about mentioning vlan is irrelevant, and instead, the issue you are having is what is described at https://ipxe.org/howto/chainloading#breaking_the_infinite_loop

**Reply by doguibnu** [reply]
Thank you 
I have no experience about ipxe but I will try study and test something!
